account_multicompany_code/models/res_company.py

```diff
@@ -1,14 +1,8 @@
-from odoo import _, api, fields, models
-from odoo.exceptions import ValidationError
+from odoo import fields, models
 
 
 class Company(models.Model):
     _inherit = "res.company"
+    _sql_constraints = [("unique_code", "UNIQUE(code)", "Code must be unique")]
```

Comment:
- Move below fields, as stated in
[the guidelines](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#id40)
- Add a trailing comma after the constraint, so it's formatted one constraint per line, i.e.:
  ```
  _sql_constraints = [
      (constraint1),
  ]
  ```